app/services/prior_authority/assessment_syncer.rb

```diff
@@ -45,6 +45,8 @@ def sync_allowances
     end
 
     def sync_primary_quote(quote, quote_data)
+      quote_data['user_chosen_cost_type'] = quote_data['cost_type']
```

Comment:
I would like to see this covered by a test. As it stands, if there was a typo in one of these strings I don't think anything would fail.

Reply:
test added, let me know if thats enough to cover it